static/app/views/onboarding/targetedOnboarding/setupDocs.tsx

```diff
@@ -1,4 +1,5 @@
-import React, {useEffect, useState} from 'react';
+import {useEffect, useState} from 'react';
+import * as React from 'react';
```

Comment:
This one's gross let's cleanup after maybe

Reply:
yeh